src/super_gradients/training/utils/checkpoint_utils.py

```diff
@@ -272,3 +272,25 @@ def load_pretrained_weights(model: torch.nn.Module, architecture: str, pretraine
                                                                               source_ckpt=pretrained_state_dict,
                                                                               solver=solver)
     model.load_state_dict(adapted_pretrained_state_dict['net'], strict=False)
+
+
+def load_pretrained_weights_local(model: torch.nn.Module, architecture: str, pretrained_weights: str):
+
+    """
+    Loads pretrained weights from the MODEL_URLS dictionary to model
+    @param architecture: name of the model's architecture
+    @param model: model to load pretrinaed weights for
+    @param pretrained_weights: path tp pretrained weights
+    @return: None
+    """
+
+    map_location = torch.device('cpu')
+
+    pretrained_state_dict = torch.load(pretrained_weights, map_location=map_location)
```

Comment:
I think we can avoid here some code duplication apearing  in load_pretrained_weights_local and load_pretrained_weights if we extract 290-296 to a helper method.

Reply:
fixed